e2e-tests/ukef/cypress/support/commands.js

```diff
@@ -25,6 +27,9 @@ Cypress.Commands.add('resetPortalUserStatusAndNumberOfSignInLinks', require('./p
 Cypress.Commands.add('enterUsernameAndPassword', require('./portal/enterUsernameAndPassword'));
 Cypress.Commands.add('completeDateFormFields', require('./portal/completeDateFormFields'));
 
+Cypress.Commands.add('makerSubmitDealForReview', makerSubmitDealForReview);
+Cypress.Commands.add('checkerSubmitDealToUkef', checkerSubmitDealToUkef);
```

Comment:
Nice work - these are really great helpers.

Could the name reflect that they're explicitly `BSS/EWCS` rather than `GEF`

If its not too much of a pain could you add these to the other deal cancellation tests :)

Reply:
@MarRobSoftwire I'll do that, but can I stick with this name for BSS and makerSubmitGefDealForReview checkerSubmitGefDealToUkef for Gef? Most of the functions are like that